python/fusion_engine_client/parsers/fast_indexer.py

```diff
@@ -13,14 +13,19 @@
 # NOTE: This needs to be larger then the biggest possible FE message. The
 # smaller it is, the faster the indexer can run, but if it's not large enough,
 # messages may be missed.
-_MAX_FE_MSG_SIZE_BYTES = 1024 * 12
+_MAX_FE_MSG_SIZE_BYTES = 1024 * 16
 
 # This has been tuned on my laptop to where increasing the read size was giving
 # diminishing returns on speed.
 _READ_SIZE_BYTES = 80 * 1024
 
 _PREAMBLE = struct.unpack('<H', MessageHeader.SYNC)
 
+# This is FileIndex._RAW_DTYPE with an additional size field. The size field is
+# used to check for overlapped messages created by `INPUT_DATA_WRAPPER` messages
```

Comment:
Instead of a specific message I'd suggest "any wrapper messages that may contain valid FusionEngine content in their payload" since there may be more than one message this applies to and since that message type is an internal message anyway.